### PR TITLE
Update to DeepTau

### PR DESCRIPTION
**Pull request by alpakpinar**
In this PR:
- Tau ID requirements updated to DeepTau
- Photon ID also updated to be compatible with nanoAODv7

Testing:
- Successfully ran on 2017 and 2018 VBF NanoAODv7 samples

### CODE REVIEW

File: bucoffea/monojet/definitions.py [hunk 1]
```diff
@@ -351,7 +351,7 @@ def setup_candidates(df, cfg):
     taus = taus[ (taus.decaymode) \
                 & (taus.pt > cfg.TAU.CUTS.PT)\
                 & (taus.abseta < cfg.TAU.CUTS.ETA) \
-                & ((taus.iso&2)==2)]
+                & (taus.iso>=2)]
```
Comment (AndreasAlbert) [inline bucoffea/monojet/definitions.py]: Is this change necessary? It seems these two statements are equivalent

File: bucoffea/config/monojet.yaml [hunk 2]
```diff
@@ -42,8 +42,8 @@ default:
       pt: 18
       eta: 2.3
     branch:
-      id: Tau_idDecayMode
-      iso: Tau_idMVAoldDM2017v2
+      id: Tau_idDecayModeNewDMs
+      iso: Tau_idDeepTau2017v2p1VSjet
```
Comment (AndreasAlbert) [inline bucoffea/config/monojet.yaml]: Lets only change VBF for now pls

**Review by AndreasAlbert — CHANGES_REQUESTED** [review]

File: bucoffea/config/monojet.yaml (hunk 2 above)
Comment (alpakpinar) [reply bucoffea/config/monojet.yaml]: Sure, will revert monojet config back to original

File: bucoffea/monojet/definitions.py (hunk 1 above)
Comment (alpakpinar) [reply bucoffea/monojet/definitions.py]: I think you're right, we don't really need the change, I'll undo it

**Comment by alpakpinar** [comment]
Pushed new changes now to revert monojet back to original (master version), removed the change in monojet/definitions.py
